### PR TITLE
Add Dot1x RADIUS-selected ACLs

### PR DESCRIPTION
**Pull request by MichaelWasher**
Add User Specifc ACL Defined by RADIUS
Add Test for User-specified ACLs
Add Test for Logout using DynACL

### CODE REVIEW

File: faucet/faucet_dot1x.py [hunk 1]
```diff
@@ -16,13 +16,13 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import functools
```
Comment (gizmoguy) [inline faucet/faucet_dot1x.py]: Remove plz

File: faucet/faucet_dot1x.py [hunk 2]
```diff
@@ -17,12 +17,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import eventlet
-
```
Comment (gizmoguy) [inline faucet/faucet_dot1x.py]: Bro add my line back

**Comment by MichaelWasher** [comment]
https://github.com/MichaelWasher/faucet/pull/5

**Comment by MichaelWasher** [comment]
Issues with travis

**Pull request closed without merging**